src/gravel/tests/conftest.py

```diff
@@ -99,10 +107,10 @@ def _get_data_contents(dir: str, fn: str):
         with open(os.path.join(dir, fn), 'r') as f:
             contents = f.read()
         return contents
-    yield _get_data_contents
+    yield _get_data_contents  # type: ignore
```

Comment:
```suggestion
    yield _get_data_contents
```

Reply:
This works if we're using the generator return, but not in any other way. While we get correctness for this function, we lose correctness (at least from the expected behavior point of view) for the rest of the tests that are using this function. We can't be telling the type checker we're yielding a function here and expect `get_data_contents` to be returning a string without calling on `next()`.